code/FulpstationCode/halloween_costumes/2020/daft_frank.dm

```diff
@@ -0,0 +1,80 @@
+//--Franklin's costumes
+//The Gold One
+/obj/item/clothing/head/hardhat/golden_punk
+	name = "Guy-Manuel Helmet"
+	desc = "Give life back to music!"
+	icon = 'icons/fulpicons/halloween_costumes/2020/daft_frank_item.dmi'
+	worn_icon = 'icons/fulpicons/halloween_costumes/2020/daft_frank_worn.dmi'
+	icon_state = "hardhat0_guy"
+	brightness_on = 1
+	on = FALSE
+	hat_type = "guy"
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0)
+	resistance_flags = null
+	flags_inv = HIDEHAIR|HIDEFACIALHAIR
+	clothing_flags = SNUG_FIT
+	flags_cover = HEADCOVERSEYES
+	dynamic_hair_suffix = ""
+
+//The Silver One
+/obj/item/clothing/head/hardhat/silver_punk
+	name = "Thomas Helmet"
+	desc = "Reminds you of touch..."
+	icon = 'icons/fulpicons/halloween_costumes/2020/daft_frank_item.dmi'
+	worn_icon = 'icons/fulpicons/halloween_costumes/2020/daft_frank_worn.dmi'
+	icon_state = "hardhat0_thomas"
+	brightness_on = 1
+	on = FALSE
+	hat_type = "thomas"
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0)
+	resistance_flags = null
+	flags_inv = HIDEHAIR|HIDEFACIALHAIR
+	clothing_flags = SNUG_FIT
+	flags_cover = HEADCOVERSEYES
+	dynamic_hair_suffix = ""
```

Comment:
names should be subtle, also why are they hardhats, this doesn't overwrite the action types either